### ISSUE TITLE
Need access to bindings

### ISSUE THREAD

**Issue by ParisNeo** [abetlen/llama-cpp-python#433]
Hi there. I am upgrading my bindings for the lord of llms tool and I now need to be able to vectorize text to embedding space of the current model. Is there a way to have access to the latent space of the model ? I input a text and get the encoder output in latent space?

Best regards

**Comment by abetlen** [comment]
Hey @ParisNeo do you mean the embeddings or the full logits? You can embed text using the create_embedding function https://llama-cpp-python.readthedocs.io/en/latest/api-reference/

**Comment by ParisNeo** [comment]
Thanks. That's what I'm searching for.
